contracts/wrapper/NameWrapper.sol

```diff
@@ -212,6 +212,28 @@ contract NameWrapper is
             (fuses & IS_DOT_ETH == 0 ||
                 expiry - GRACE_PERIOD >= block.timestamp);
     }
+    
+    /**
+     * @notice Check if the address is the owner or approved by owner.
+     * @param owner The owner of the name.
+     * @param fuses The fuses of the name.
+     * @param expiry The expiry of the name.
+     * @param addr The address address to check permissions for.
+     * @return whether or not is owner or approved.
+     */
+
+    function canModifyNameWithData(address owner, uint32 fuses, uint64 expiry, address addr)
```

Comment:
```suggestion
    function _canModifyNameWithData(address owner, uint32 fuses, uint64 expiry, address addr)
```

Reply:
Can you use this internal function inside `canModifyName`?

Reply:
Ok, but it will just cost more gas.

Reply:
> @jefflau Ok, I committed your comment change.

Okay, just need the internal function changes resolved and prefixed with an underscore. And move it down to the internal functions section of the contract

Reply:
ok